corehq/apps/ota/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DemoUserRestore',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('demo_user_id', models.CharField(default=None, max_length=255, db_index=True)),
```

Comment:
Can you split this into 2 so that the `CreateModel` op doesn't create the index and then a second opp adds the index. That way we won't get the parttern ops index which we don't want.